src/OrchardCore.Modules/OrchardCore.Workflows/Views/Workflow/Details.cshtml

```diff
@@ -94,8 +94,8 @@
     <a class="btn btn-danger" data-message="@T["Are you sure you want to delete this workflow instance?"]" data-url-af="RemoveUrl UnsafeUrl" asp-action="Delete" asp-route-id="@Model.Workflow.Id">@T["Delete"]</a>
 </div>
 
-<style asp-src="~/OrchardCore.Workflows/Styles/jsplumbtoolkit-defaults.min.css" debug-src="~/OrchardCore.Workflows/Styles/jsplumbtoolkit-defaults.css" cdn-src="https://cdnjs.cloudflare.com/ajax/libs/jsPlumb/2.5.5/css/jsplumbtoolkit-defaults.min.css" debug-cdn-src="https://cdnjs.cloudflare.com/ajax/libs/jsPlumb/2.5.5/css/jsplumbtoolkit-defaults.css" version="2.5.5"></style>
+<style asp-name="jsplumbtoolkit-defaults"></style>
 <style asp-src="~/OrchardCore.Workflows/Styles/orchard.workflows-viewer.min.css" debug-src="~/OrchardCore.Workflows/Styles/orchard.workflows-viewer.css"></style>
 <script asp-name="bootstrap" version="4" at="Foot"></script>
-<script asp-name="jsplumb" depends-on="jQuery" asp-src="~/OrchardCore.Workflows/Scripts/jsplumb.min.js" debug-src="~/OrchardCore.Workflows/Scripts/jsplumb.js" cdn-src="https://cdnjs.cloudflare.com/ajax/libs/jsPlumb/2.5.5/js/jsplumb.min.js" debug-cdn-src="https://cdnjs.cloudflare.com/ajax/libs/jsPlumb/2.5.5/js/jsplumb.js" version="2.5.5" at="Foot"></script>
+<script asp-name="jsplumb" at="Foot"></script>
```

Comment:
Do you mean this one is rendered twice?

Probably you don't need to include it at all, because the script below `depends-on` it.

But I wouldn't have expected it to be registered twice because of that. maybe an issue in the resource dependencies

Reply:
Removing jsplumb script (depends-on will include it) works.